cpp/cmake_modules/FindArrowPython.cmake

```diff
@@ -15,7 +15,7 @@
 # specific language governing permissions and limitations
 # under the License.
 
-# - Find Arrow Python (arrow/python/api.h, libarrow_python.a, libarrow_python.so)
+# - Find Arrow Python (python/pyarrrow/src_arrow/api.h, libarrow_python.a, libarrow_python.so)
```

Comment:
This is a needless change.
The path shows what value can be used for `#include` not where `api.h` is placed in source directory.